gke-deploy.sh

```diff
@@ -17,6 +17,10 @@
 
 set -eu
 
+cd chart/gateway
+helm dependency update
+cd -
+
 if [[ "${ENVIRONMENT:-}" = 'production' ]]; then
```

Comment:
Where does the environment variable normally get set?  Would be good to add a comment.